### PR TITLE
Added frontend-app-component-footer-mitol

### LINKED ISSUE #411: Update Build Pipelines to Include frontend-app-footer-mitol
## Description

Override concourse build pipelines to include fronend-app-footer-mitol as an alias for frontend-app-footer

## Related documents.

- https://edx.readthedocs.io/projects/edx-developer-docs/en/latest/developers_guide/micro_frontends_in_open_edx.html#deployment

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
# Description
Replaced frontend-component-footer with @mitol/frontend-component-footer-mitol using npm aliases  and added env variables.

## which are relevant tickets.
Closes #411

### CODE REVIEW

**Comment by blarghmatey** [comment]
Looks good :+1:

**Review by blarghmatey — APPROVED** [review]

File: pipelines/edx/mfe-frontend-app-learning-qa.yml [hunk 1]
```diff
@@ -59,13 +59,24 @@ jobs:
         SUPPORT_URL: https://mitx-micromasters.zendesk.com/hc/en-us/requests/new
         USER_INFO_COOKIE_NAME: edx-user-info
         SESSION_COOKIE_DOMAIN: .mitxonline.mit.edu
+        ABOUT_US_URL: https://mitxonline.mit.edu/about-us/
```
Comment (blarghmatey) [inline pipelines/edx/mfe-frontend-app-learning-qa.yml:62]: This should be for rc.mitxonline.mit.edu

File: pipelines/edx/mfe-frontend-app-learning-qa.yml [hunk 2]
```diff
@@ -59,13 +59,24 @@ jobs:
...
+        PRIVACY_POLICY_URL: https://mitxonline.mit.edu/privacy-policy/
+        HONOR_CODE_URL: https://mitxonline.mit.edu/honor-code/
+        TERMS_OF_SERVICE_URL: https://mitxonline.mit.edu/terms-of-service/
+        Contact: mailto:mitxonline-support@mit.edu
+        SUPPORT_CENTER_URL: https://xpro.zendesk.com/hc/en-us
```
Comment (blarghmatey) [inline pipelines/edx/mfe-frontend-app-learning-qa.yml:67]: Is this right?

**Review by blarghmatey — CHANGES_REQUESTED** [review]
Since this pipeline is specific to QA we should make the links point to the RC site.

File: pipelines/edx/mfe-frontend-app-learning-qa.yml [hunk 3]
```diff
@@ -59,13 +59,24 @@ jobs:
         SUPPORT_URL: https://mitx-micromasters.zendesk.com/hc/en-us/requests/new
         USER_INFO_COOKIE_NAME: edx-user-info
         SESSION_COOKIE_DOMAIN: .mitxonline.mit.edu
+        ABOUT_US_URL: https://mitxonline.mit.edu/about-us/
+        PRIVACY_POLICY_URL: https://mitxonline.mit.edu/privacy-policy/
+        HONOR_CODE_URL: https://mitxonline.mit.edu/honor-code/
+        TERMS_OF_SERVICE_URL: https://mitxonline.mit.edu/terms-of-service/
+        Contact: mailto:mitxonline-support@mit.edu
```
Comment (pdpinch) [inline pipelines/edx/mfe-frontend-app-learning-qa.yml:66]: is `Contact` the right key?

**Review by blarghmatey — APPROVED** [review]

**Comment by blarghmatey** [comment]
:+1:

File: pipelines/edx/mfe-frontend-app-learning-qa.yml (hunk 3 above)
Comment (Wassaf-Shahzad) [reply pipelines/edx/mfe-frontend-app-learning-qa.yml:66]: Yes, I missed the missed the CAPS on the key somehow. I will update it if we need to change the code after QA test.

**Pull request merged**